pkg/cvo/internal/operatorstatus.go

```diff
@@ -96,6 +96,11 @@ func (b *clusterOperatorBuilder) WithModifier(f resourcebuilder.MetaV1ObjectModi
 
 func (b *clusterOperatorBuilder) Do(ctx context.Context) error {
 	os := readClusterOperatorV1OrDie(b.raw)
+
+	// add cluster operator's start time if not already there
+	if !payload.COUpdateStartTimesHasName(os.Name) {
+		payload.COUpdateStartTimesAdd(os.Name)
```

Comment:
this will create two lock/release cycles.  Can we make this a single `payload.COUpdateStartTimesEnsureName` or something that is add-if-missing?

Reply:
Good catch. I had it like this when I originally used a RW lock, which had its own issues, so yes, make sense to only lock once.

Reply:
Changed in [9298cb3](https://github.com/openshift/cluster-version-operator/pull/486/commits/9298cb355153d4145154c5c37083eda8a01be7d3)